songsuro/evaluation/melody_evaluator.py

```diff
@@ -0,0 +1,124 @@
+import torch
+import torchcrepe
+from sklearn.metrics import f1_score
+from songsuro.preprocess import load_audio, make_spectrogram, make_mel_spectrogram
+
+
+class MelodyLoss:
+	def __init__(self, wav_gt_path=None, wav_gen_path=None):
+		if wav_gt_path is not None and wav_gen_path is not None:
+			self.wav_gen = load_audio(wav_gen_path)
+			self.wav_gt = load_audio(wav_gt_path)
```

Comment:
Why load audio again?
In validation and test process we can get ground truth mel-spectorgram and generated audio waveform as torch tensor. 
If we save it to wav file and load it again it will be super inefficient.